onnxruntime/core/graph/contrib_ops/contrib_ops/bert_defs.cc

```diff
@@ -250,7 +250,7 @@ Enforce no repetition of n-grams. Scores are set to `-inf` for tokens that form
 )DOC";
 
 ONNX_MS_OPERATOR_SET_SCHEMA(NGramRepeatBlock, 1,
-                            OpSchema().SetDoc(NGramRepeatBlock_ver1_doc).Attr("ngram_size", "The NGram size.", AttributeProto::INT).Input(0, "input_ids", "2D input tensor with shape (batch_size, sequence_length)", "Tid").Input(1, "scores", "2D input tensor with shape (batch_size, vocab_size)", "T").Output(0, "scores_out", "2D output tensor with shape (batch_size, vocab_size)", "T").TypeConstraint("Tid", {"tensor(int64)"}, "Constrain indices to integer types").TypeConstraint("T", {"tensor(float)"}, "Constrain scores input and output types to float tensors.").TypeAndShapeInferenceFunction([](ONNX_NAMESPACE::InferenceContext& ctx) {
+                            OpSchema().SetDoc(NGramRepeatBlock_ver1_doc).Attr("ngram_size", "The NGram size.", AttributeProto::INT).Attr("banned_no_further_than", "Banned no repeat ngram tokens with no further than a certain number of look back.", AttributeProto::INT).Input(0, "input_ids", "2D input tensor with shape (batch_size, sequence_length)", "Tid").Input(1, "scores", "2D input tensor with shape (batch_size, vocab_size)", "T").Output(0, "scores_out", "2D output tensor with shape (batch_size, vocab_size)", "T").TypeConstraint("Tid", {"tensor(int64)"}, "Constrain indices to integer types").TypeConstraint("T", {"tensor(float)"}, "Constrain scores input and output types to float tensors.").TypeAndShapeInferenceFunction([](ONNX_NAMESPACE::InferenceContext& ctx) {
```

Comment:
Consider renaming the attribute, and rephrasing the description. I'm thinking something like `recency_length`: "Only look at the last `recency_length` tokens when enforcing no repetition of n-grams. [value range] [default value and behavior] etc".

Reply:
Done